src/app/TERMS.md

```diff
@@ -0,0 +1,252 @@
+
+## General Information
+
+By accessing or using the Services, you agree that you have read, understood and accept all of the terms and conditions contained in this Agreement as well as our Privacy Policy. You also represent that you have the legal authority to accept these Terms on behalf of yourself and any party you represent in connection with your use of the Services. If you are an individual who is entering into these Terms on behalf of an entity, you represent and warrant that you have the power to bind that entity, and you hereby agree on that entity’s behalf to be bound by these Terms, with the terms “you”, and “your” applying to you, that entity, and other users accessing the Services on behalf of that entity.
+
+We may, in our sole and absolute discretion, without liability to you or any third-party, refuse to let you use of one or more of the Services. Such actions may be taken as a result of a number of factors, including without limitation legal/regulatory demand, or your violation of the terms of this Agreement. We may also temporarily suspend your access to the Services if a technical problem so requires.
+
+## Security of your Account and Wallet
+
+We are not an insured financial institution. As a result, unlike funds held in most banks or credit unions, amounts displayed through the Services are not insured against loss.
+
+Furthermore, DeXter is a non-custodial platform. As such, we do not know nor have access to the private keys to the digital asset wallets you use in connection with the Services (“Wallets”). Therefore, you are exclusively responsible for maintaining the confidentiality and security of your Accounts or devices you use to access the Services. You are also responsible for maintaining the security of the digital asset wallets you utilize in connection with the Services, and for ensuring that no unauthorized person has access to your wallet files, its private keys, or any device that you utilize in connection with the Services.
+
+We will not be liable for any loss or damage arising from your failure to protect your Account or your Wallets.
+
+It is your sole responsibility to provide accurate information for creation and maintenance of your account. You are solely responsible for ensuring the accuracy and completeness of all information and materials that you provide to use in connection with your use of the Site or Services. This includes, but is not limited to, all information and materials that you provide to us in connection with background checks and identity verification. You represent and warrant that (i) all such information and materials are true, accurate, and complete in all respects, comply with applicable law and do not violate or infringe any third-party rights; and (ii) you will immediately notify us about, and correct, and inaccuracy in any such materials or information.
```

Comment:
This isn't relevant i don't think

Reply:
Sure.  Definitely can remove the bits we don't need.  Converted this into a draft since its no rush and others can suggest as well.